packages/contentstack-bulk-publish/package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "@contentstack/cli-cm-bulk-publish",
   "description": "Contentstack CLI plugin for bulk publish actions",
-  "version": "1.9.0",
+  "version": "1.9.1",
```

Comment:
@sunil-lakshman its a new feature not a bug fix or a patch, for feature updates we have to bump the minor version,  this should reflects in the contentstack package.json as well cli will get minor update